python/cuda_cooperative/cuda/cooperative/experimental/block/_block_load_store.py

```diff
@@ -36,6 +39,49 @@
 
 
 def load(dtype, threads_in_block, items_per_thread=1, algorithm="direct"):
+    """Creates an operation that performs a block-wide load.
+
+    Returns a callable object that can be linked to and invoked from device code. It can be
+    invoked with the following signatures:
+
+    - `(src: numba.types.Array, dest: numba.types.Array) -> dtype`: Each thread loads
```

Comment:
important: I don't see load and store return anything
```suggestion
    - `(src: numba.types.Array, dest: numba.types.Array) -> None`: Each thread loads
```